test/e2e/performanceprofile/functests/utils/cgroup/v1/v1.go

```diff
@@ -0,0 +1,90 @@
+package v1
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/cgroup/controller"
+	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/cgroup/runtime"
+	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/pods"
+)
+
+type ControllersManager struct {
+	client    client.Client
+	k8sClient *kubernetes.Clientset
+}
+
+func NewManager(c client.Client, k8sClient *kubernetes.Clientset) *ControllersManager {
+	return &ControllersManager{client: c, k8sClient: k8sClient}
+}
+
+func (cm *ControllersManager) CpuSet(ctx context.Context, pod *corev1.Pod, containerName, runtimeType string) (*controller.CpuSet, error) {
```

Comment:
We are passing runtimeType but we are not using this variable in CpuSet function